sdk-core/src/main/java/com/sportradar/unifiedodds/sdk/di/GeneralModule.java

```diff
@@ -193,15 +187,22 @@ private Unmarshaller provideMessageUnmarshaller() {
         }
     }
 
-    /**
-     * Provides the {@link Deserializer} used to deserialize sports API xmls
-     *
-     * @return The {@link Deserializer} instance to be registered with the DI container
-     */
-    @Provides @Named("SportsApiJaxbDeserializer")
-    private Deserializer provideSportsApiJaxbDeserializer() {
-        return new DeserializerImpl(sportsApiJaxbContext);
-    }
+  @Provides
```

Comment:
This was Mark ensuring just one XML Marshaller wasn't used, I think he bound one to each thread